gypbuild.js

```diff
@@ -29,6 +31,11 @@ function runGyp (opts, target, cb) {
   }
   if (opts.debug) args.push('--debug')
 
+  // Since electron and node are reusing the versions now (fx 6.0.0) and
+  // node-gyp only uses the version to store the dev files, they have started
+  // clashing. To work around this we explicitly set devdir to tmpdir/runtime(/target)
+  args.push('--devdir=' + path.join(os.tmpdir(), 'prebuild', opts.runtime || 'node'))
```

Comment:
Hm, when running tests locally I don't see this directory being created.

Reply:
Ah. Need to modify this instead:

https://github.com/prebuild/prebuild/blob/4ac850fc87bf9baef097701f7f770dffaec548e9/gyp.js#L28